### DISCUSSION TITLE
Recommended way to respect 2nd clause of BSD license?

### DISCUSSION THREAD

**Discussion by ckcr4lyf** [JulianSchmid/etherparse#38] [Q&A] [answered]
I am planning to use this library in my project (https://github.com/ckcr4lyf/sni-log) .

While including the notice from https://github.com/JulianSchmid/etherparse/blob/aebf1a53097928ba46c097b218e32bb5e084816f/LICENSE in the source is easy, do you have any recommendation for including it in binary form?

I am planning to make a CLI app, so I am not sure the best way to do so...

Thanks for this lib btw!

**Comment by JulianSchmid** [comment]
Hi,

I don't really have an opinion and never really thought about it. It was not really my intention to introduce these kind of problems when I chose the BSD-3-clause license.

I could just bypass the hole issue publishing a new minor version of the library that also offers the MIT & Apache (or similar) or replaces the BSD license completely. But I have to look into it over the weekend.

Greets
Julian

**Reply by ckcr4lyf** [reply]
Thanks for the reply. I'll also look around a bit to see if other people are doing it in a pure CLI app

**Comment by JulianSchmid** [comment, marked as answer]
Hi @ckcr4lyf ,

I just released https://github.com/JulianSchmid/etherparse/releases/tag/v0.13.0 under the MIT or Apache 2.0 license (your choice). That should remove the requirement for the inclusion of the license in the binary artifacts.

Greets
Julian

**Reply by ckcr4lyf** [reply]
Thanks, that works great ^_^
